scipy/stats/_continuous_distns.py

```diff
@@ -3554,6 +3559,66 @@ def fit(self, data, *args, **kwds):
             fshape_s = fshape_n / fscale
         return fshape_s, floc, fscale
 
+    def _ppf_isf(self, q, mu, upper):
+        """ calculate ppf or isf depending on the value of `upper`.
+        If upper is True, then compute isf, else ppf. This implementation is
+        based on Gyner & Smyth (2016) and uses newton's method with carefully
+        selected starting points that guarantee convergence regardless of the
+        probability."""
+
+        def f(a, b, c, d, e):
+            """approximates (x0 - norm_cdf) / pdf when abs(x0 - norm_cdf) < 1e-5"""
+            return a * np.exp(c + np.log1p(-0.5 * a) - d)
+
+        def f2(a, b, c, d, e):
```

Comment:
Does not depend on `a`

Reply:
Its needed by the lazywhere function since it expects the functions `f` and `f2` to have equal size of arguments. What do you suggest I do to get around that?